Controllers/Customer_UsersController.cs

```diff
@@ -26,6 +26,7 @@ public Customer_UsersController(RecitopiaDBContext recitopiaDbContext)
         public async Task<IActionResult> Index()
         {
             //BUILD VIEW FOR ANGULARJS RENDERING
+            //For some reason, I am unable to recreate this quagmire in LINQ format.  most likely to do with Models
```

Comment:
Where this comment is, I believe I need to update some models so that LINQ can JOIN the tables.

Reply:
What is the purpose of having customer users?